plugins/linuxv2/model/interfaces/interfaces.proto

```diff
@@ -0,0 +1,33 @@
+syntax = "proto3";
+
+package interfaces;
+
+import "github.com/ligato/vpp-agent/plugins/linuxv2/model/namespace/namespace.proto";
+
+enum LinuxInterfaceType {               /* Available interface types */
+    VETH = 0;
+    AUTO_TAP = 1;
+};
+
+message LinuxInterface {
+    string name = 1;                    /* Logical interface name unique across all configured interfaces (mandatory) */
+    LinuxInterfaceType type = 2;        /* Interface type (mandatory) */
+    bool enabled = 3;
+    repeated string ip_addresses = 4;   /* IP address in format <ipAddress>/<ipPrefix> */
+    string phys_address = 5;            /* MAC address */
+    uint32 mtu = 6;                     /* Maximum transmission unit value */
+    string host_if_name = 7;            /* Name of the interface in the host OS. If not set, the host name
```

Comment:
Couldn't we get rid of this and use just `name`?

Reply:
Nope, you can have interfaces with the same host name, but in different namespaces - like in K8s all pods must have eth0 interface.